src/routes/docs/tutorials/flutter-auth/step-3/+page.markdoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: tutorial
+title: Initialize SDK
+description: Add Authentication to a Flutter project using Appwrite.
+step: 3
+---
+Before you can use Appwrite, you need to instanciate the Appwrite `Client` class with the project ID and endpoint. 
+This tells the SDK where your Appwrite project is hosted and which one to connect to.
+
+The client is then used to initialize services like `Databases` and `Account`, so they all point to the same Appwrite project.
+
+You can do this by instantiating the services you need in a file like `lib/config/appwrite.dart` and **exporting the instances**.
+
+```dart
+// lib/config/appwrite.dart
+import 'package:appwrite/appwrite.dart';
+
+import 'package:my_flutter_project/constants.dart' as constants;
+
+class Appwrite {
+  static final Appwrite instance = Appwrite._instance();
+
+  late final Client client;
+
+  late final Account account;
+
+  Appwrite._instance() {
+    client = Client()
+        .setEndpoint(constants.appwriteEndpoint)
+        .setProject(constants.appwriteProjectID)
+        .setSelfSigned(status: true);
```

Comment:
We're gonna suggest they do this against cloud so the appwrite endpoint should default to cloud and setSelfSigned should not be set :D